addon/session-stores/cookie.js

```diff
@@ -143,8 +166,7 @@ export default BaseStore.extend({
   },
 
   _read(name) {
-    let value = document.cookie.match(new RegExp(`${name}=([^;]+)`)) || [];
-    return decodeURIComponent(value[1] || '');
+    return this.get('_cookies').read(name) || '';
```

Comment:
This is actually only generating an error when using the adaptive store. Of course that should work as well as the adaptive store should simply fallback to the cookie store and work out of the box (although the session would not actually be transported between the client and server then as the client would use `localStorage`). It should certainly not lead to an error though.